packages/components/src/components/data-list/data-list.scss

```diff
@@ -93,6 +93,25 @@
     }
 }
 
+.statement__row--detail-text-mobile {
+    width: calc(100vw - 32px);
+    height: 100%;
+    text-align: left;
+    padding: 6px 16px;
```

Comment:
Try to use `rem` and replace all `px`, make sure all of them follow the design

Reply:
The design is using `px`. Should I just convert the values to `rem`?

Reply:
yes